pandas/tests/indexes/common.py

```diff
@@ -514,12 +514,12 @@ def test_union_base(self, indices):
 
     @pytest.mark.parametrize("sort", [None, False])
```

Comment:
not for this PR, but we have a fixture in tests.indexes.interval.test_setops for this

Reply:
Created a follow-up issue: #32183